crates/bestool/src/actions/tamanu.rs

```diff
@@ -120,3 +122,42 @@ pub fn find_existing_version() -> Result<Version> {
 		.pm2_env
 		.version)
 }
+
+#[cfg(not(windows))]
+fn find_postgres_bin(name: &str) -> Result<&str> {
```

Comment:
These should be gated with a new feature e.g. `common-pg` so that they don't show up as dead code if the tamanu-psql and -backup features are disabled